src/core/feed_forward.jl

```diff
@@ -271,14 +291,22 @@ function feed_forward_update(sync::Chron,
                             param_reference::UpdateRef{JuMP.VariableRef},
                             param_array::JuMPParamArray,
                             to_stage::Stage,
-                            from_stage::Stage) where Chron <: AbstractChronology
-    !(to_get_execution_count(stage) % sync.to_steps == 0) && return
-
-    var_count = to_get_execution_count(stage) ÷ sync.to_steps
-
-    for device_name in axes(param_array)[1]
+                            from_stage::Stage,
+                            to_interval::T) where {Chron <: AbstractChronology,
+                                                   T<:Dates.TimePeriod}
+
+    execution_count = get_execution_count(to_stage) 
+    from_stage_res = PSY.get_forecasts_resolution(PSI.get_sys(from_stage))
+    to_stage_res = PSY.get_forecasts_resolution(PSI.get_sys(to_stage))
+    for device_name in axes(param_array)[1] , time in axes(param_array)[2]
+        remainder = (time*to_stage_res + (execution_count)*to_interval) % from_stage_res
+        if remainder == Dates.Millisecond(0)
+            var_count = Int((time*to_stage_res + (execution_count)*to_interval) / from_stage_res )
+        else
+            var_count = Int(ceil((time*to_stage_res + (execution_count)*to_interval) / from_stage_res))
```

Comment:
Why are we using ceil here? We need to make sure the counts are consistent with the inputs. These hacks are what I try to avoid by having extra inputs and guaranteeing that the counts are consistent. Please avoid the use of ceil in these cases.